reports/technical_report/5_conclusions.md

```diff
@@ -60,16 +60,18 @@ Finally, we would like to incorporate additional secondary data into our analysi
 ### Micro analysis
 
 There are two principle significant investments required to improve the data quality of Covid-19 notices.
-Firstly, the data collected by Glass must be more reliable and complete: notices that are truncated must not be truncated; notices containing snippets from different sections of a businesses website must be separated; the time and date of collection must be provided rather than just a month.
+Firstly, the data collected by Glass must be more reliable and complete: notices that are truncated must not be truncated; notices containing snippets from different sections of a businesses website must be separated; and the time and date of collection must be provided rather than just a month.
 Unfortunately improved data collection would have to start again and would not be able to recover historic information, thus losing a key feature of the data.
 Secondly, better processing of the notices must be performed such that irrelevant notices are filtered out and relevant notices are classified into several categories such as "Business as usual", "Temporary closure", "Permanent closure", "Adaptation", "Partial closure" etc.
-This would likely require hand-annotation of the relevant sections of a subset of notices; training a model on these labels; extrapolating to the rest of the notices using the model; and performing a validation study on the results.
+This would require hand-annotation of a subset of notices; training a model on these labels; extrapolating to the rest of the notices using the model; and performing a validation study on the results.
 [Recent analysis](https://datasciencecampus.ons.gov.uk/extracting-text-data-from-business-website-covid-19-notices/)
  by the ONS has already piloted this hand labelling approach.
-While they find some promise in this approach; they are cautious about the biases of this approach.
+While they find some promise in this approach, they are cautious about the biases of this approach.
 
-Given the myriad of issues with the Covid notice data, the fact that the ONS is also exploring similar lines of enquiry, and the ONS has a competitive edge with access to microdata:
-there is probably more value in investing effort into social-media data over notice data.
+There is probably more value in investing effort into social-media data over notice data given:
+ the myriad of issues with the Covid notice data,
+ the fact that the ONS is also exploring similar lines of enquiry,
+ and the ONS has a competitive edge with its access to microdata.
 A potential scale-up option for the twitter data would be to put the twitter dataset into Elasticsearch to generate a search engine for the tweets of Scottish businesses which could be queried by policymakers to obtain lists of tweets relating to the query and interactive visualisations communicating the trends across 
```

Comment:
I would reorder this paragraph and begin talking about scaling up in data collection and leave the searchable / explorable output for next section where we talk about visualisation options. Other potential research avenues I would consider:
* Analyse sectoral and geographical coverage
* Collect data for all the UK in order to compare the situation in Scotland: are there specific sectors / locations particularly vocal about issues with Covid-19?
* More sophisticated analysis of text in tweets